packages/block-editor/src/components/colors/utils.js

```diff
@@ -60,14 +60,32 @@ export function getColorClassName( colorContextName, colorSlug ) {
 		return undefined;
 	}
 
+	// In the past, we used lodash's kebabCase to process slugs.
+	// By doing so, this method also accepted and converted non string values
+	// into strings. Some plugins relied on this behavior.
+	if ( 'string' !== typeof colorSlug ) {
+		colorSlug = String( colorSlug );
+		// eslint-disable-next-line no-console
+		console.warn( 'The color slug should be a string.' );
+	}
```

Comment:
What non-string value would be permissible here? Numbers, for example, don't seem to make sense here, do they?

Reply:
Same comment as https://github.com/WordPress/gutenberg/pull/32720#discussion_r652778688 I think the issue is the same in both cases: that consumers of the API expected much more than we thought we offered. That there isn't a specific report yet doesn't mean there can't be some.